noxfile.py

```diff
@@ -30,7 +30,7 @@ def pip_compile(session: nox.Session, req: str):
     # Use --upgrade by default unless a user passes -P.
     args = list(session.posargs)
     if not any(
-        arg.startswith("-P") or arg.startswith("--upgrade-package") for arg in args
+        arg.startswith(("-P", "--upgrade-package", "--no-upgrade")) for arg in args
     ):
```

Comment:
Is this related to this pr? I'm fuzzy on nox stuff but not immediately seeing how this relates to the mkdocs redirects.

Reply:
Yeah it probably should go into a separate PR. I had to add the `--no-upgrade` flag here so that only the `mkdocs-redirects` package would get added to the requirements. Otherwise the nox session upgrades all the other packages.

Reply:
Moved to https://github.com/ansible/ecosystem-documentation/pull/32